.github/workflows/publish_connectors.yml

```diff
@@ -53,6 +54,7 @@ jobs:
           gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
           github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
           metadata_service_gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
+          sentry_dsn: "https://f2e9aeb6b24b4024b09679201c7f563e@o1009025.ingest.sentry.io/4505596642983936"
```

Comment:
not a secret?

Reply:
It's not, but if you have admin access and can help setting it up as one we can get rid of this duplication 😄 @evantahler

Reply:
Done! `SENTRY_CONNECTOR_DAGSTER_DSN` is what you want now

Reply:
It should be `SENTRY_AIRBYTE_CI_DSN` not `DAGSTER` 😄 will change as I've admin access.

Reply:
@pedroslopez I pushed the change.